shell/platform/android/io/flutter/embedding/engine/FlutterJNI.java

```diff
@@ -121,7 +121,7 @@ public static native void nativeInit(
 
   // TODO(mattcarroll): add javadocs
   @UiThread
-  public static native boolean nativeGetIsSoftwareRenderingEnabled();
+  public native boolean nativeGetIsSoftwareRenderingEnabled();
```

Comment:
Was this meant to be part of this change?

Reply:
I see now.

This seems to indicate that most of these methods shouldn't be static then, right?